server/helpers/authHelpers.js

```diff
@@ -0,0 +1,60 @@
+import jwt from 'jsonwebtoken';
+import { config } from 'dotenv';
+import sgMail from '@sendgrid/mail';
+import { User } from '../sequelize/models/index'; 
+import bcrypt from 'bcrypt';
+
+
+config();
+sgMail.setApiKey(process.env.SENDGRID_API_KEY);
+
+class authHelper {
+  /**This class is to find if the user's email exists or not exists
+   * @param {string} attr data from the user table.
+   * @param {string} val value to be found
+   * @returns {object} email of the user
+  */
+
+  static async gotUser(attr, val) {
```

Comment:
Consider updating the method name to **findUser** instead.

Reply:
Done